src/main/java/org/folio/dew/batch/bursarfeesfines/BursarExportJobConfig.java

```diff
@@ -37,20 +40,34 @@ public class BursarExportJobConfig {
   @Bean
   public Job bursarExportJob(
     Step prepareContext,
-    Step exportStep,
+    @Qualifier("exportStepRegular") Step exportStepRegular,
```

Comment:
I don't think the `@Qualifier` is needed here, since the parameter name is the same as the step/method name.  It's only needed when ambiguous.

(This may be wrong, I'm about 85% sure tho)